gce/survival-training/gce/startup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+GCE_USER=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/gceUser -H "Metadata-Flavor: Google")
+TRAINER_REPO=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/trainerRepo -H "Metadata-Flavor: Google")
+TRAINER_MODULE=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/trainerModule -H "Metadata-Flavor: Google")
+
+JOB_DIR=$(curl http://metadata.google.internal/computeMetadata/v1/instance/attributes/jobDir -H "Metadata-Flavor: Google")
+
```

Comment:
Add some comments in the script and also around the variables

Reply:
@puneith : Added, please have a look

Reply:
Great. Few additional comments around ENVs will be really helpful.

Reply:
Done, please have a look.